Zinc/Private/ZincRepo.m

```diff
@@ -34,8 +34,10 @@
 #define DOWNLOADS_DIR @"zinc/downloads"
 #define REPO_INDEX_FILE @"repo.json"
 
-static NSMutableDictionary* _ReposByURL;
+static const float kContentBundleFlushLimitInMegabytes = 100.f;
+static NSString * const kContentBundlePrefix = @"com.wonder.content";
```

Comment:
yeah, ideally you'd pass in the values kinda like we do for the source urls, where we do `[zincRepo addSourceURL:sourceURL];`. you could call it temporary bundles or something of the sort so people know they can be deleted at any time. then provide zinc with the list of prefixes, and the task would delete the items that match the prefix.